adoc/reference-webui-systems.adoc

```diff
@@ -2690,7 +2693,7 @@ By default, the following packages are added: [package]#rhncfg#, [package]#rhncf
 If you select [guimenu]``Virtualization Host`` you automatically get the following package: [package]#rhn-virtualization-host#.
 
 Adding the [package]#osad# package makes sense to execute scheduled actions immediately after the schedule time.
-When the activation key is created, you can add packages with selecting the key (menu:Software[Activation Keys]), then on the activation key details tab, go for the [guimenu]``Packages`` subtab and add [package]#osad#.
+When the activation key is created, you can add packages with selecting the key (menu:Main Menu[Systems > Activation Keys]), then on the activation key details page, go for the [guimenu]``Packages`` tab and add [package]#osad#.
```

Comment:
" ... add packages by selecting ... "
Not sure what is meant by "go for" here, perhaps "click"?

Reply:
yes